### PR TITLE
[DOCS] EQL: Document `case_sensitive` param

### PR DESCRIPTION
**Pull request by jrodewig**
Relates to #56404

### CODE REVIEW

File: docs/reference/eql/search.asciidoc [hunk 1]
```diff
@@ -213,3 +213,33 @@ GET /sec_logs/_eql/search
...
+[[eql-search-case-sensitive]]
+=== Run a case-sensitive EQL search
+
+By default, matching for EQL queries is case-insensitive. You can use the EQL
+search API's `case_sensitive` parameter to toggle case sensitivity on or off.
+
+.*Example*
+[%collapsible]
+====
+The following request searches contains a query that matches `process` events
```
Comment (astefan) [inline docs/reference/eql/search.asciidoc:227]: `The following search request contains a query....` maybe? I am not sure I understand "The following request searches contains a query..."

**Review by astefan — APPROVED** [review]
LGTM. Thanks.
Left a suggestion.

File: docs/reference/eql/search.asciidoc (hunk 1 above)
Comment (jrodewig) [reply docs/reference/eql/search.asciidoc:227]: Thanks for catching this. That's a typo on my part. Your suggestion is correct.

**Comment by jrodewig** [comment]
### Backport commits
master 19699af
7.x c50f86f

**Pull request merged**